libc/math/frac.c

```diff
@@ -0,0 +1,303 @@
+/*
+ * Phoenix-RTOS
+ *
+ * POSIX.1-2017 standard library functions tests
+ *
+ * HEADER:
+ *    - math.h
+ *
+ * TESTED:
+ *    - modf(), fmod()
+ *    - ceil, floor()
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Adam Debek
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <math.h>
+#include <limits.h>
+#include <float.h>
+#include <time.h>
+#include <stdlib.h>
+#include <errno.h>
+
+#include "common.h"
+#include <unity_fixture.h>
+
+static double test_ceilAndFloor(double x, int type)
```

Comment:
Need to use `and` in function name is a good hint something should be changed.

My suggestion - provide separate `ceil` and `floor` and share common code